cloud/gcp/modules/compute/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "google_cloud_run_service" "civiform_application_run_service" {
+  name     = "cloudrun-civiform"
+  location = var.region
+
+  ##TODO(@ktoor): Change to correct docker image. Will need to push the image to gcr through some mechanism.
+  template {
+    spec {
+      containers {
+        image = "us-docker.pkg.dev/cloudrun/container/hello"
+        resources {
+            requests = {
+                "cpu"    = "1"
+                "memory" = "512Mi"
+            }
+            limits = {
+                cpu      = "4"
+                "memory" = "7182Mi"
+            }
+        }
+        ports {
+          name = "http1"
+          container_port = var.http_port
+        }
+        env {
+          name = "DB_USERNAME"
+          value = var.application_service_account_email
+        }
+        env {
+          name = "REGION"
+          value = var.region
+        }
+        env {
+          name = "GCS_BUCKET_NAME"
+          value = var.bucket_name
+        }
+        // CloudSql JDBC connector connects a little differently(see https://cloud.google.com/sql/docs/postgres/connect-run)
```

Comment:
```suggestion
        // CloudSql JDBC connector connects a little differently
        // See https://cloud.google.com/sql/docs/postgres/connect-run
```

Reply:
A little differently from what (please clarify in comment)?

Reply:
Done